addon/services/ajax.js

```diff
@@ -73,6 +73,8 @@ const {
   ```
 
 **/
+const hasDOM = typeof document !== 'undefined';
```

Comment:
@stefanpennar I can check for the existence of `najax`  (which I believe was rwjblue's original suggestion)